tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -30,16 +30,19 @@ def check_ceph_used_space():
     """
     Check for the used space in cluster
     """
-
-    # ToDo: after running IO's or deleting pod & pvc takes some time
-    time.sleep(20)
-    ct_pod = pod.get_ceph_tools_pod()
-    ceph_status = ct_pod.exec_ceph_cmd(ceph_cmd="ceph status")
-    assert ceph_status is not None
-    used = ceph_status['pgmap']['bytes_used']
-    GB = (1024 * 1024 * 1024)
-    used_in_gb = used / GB
-    return used_in_gb
+    end_time = datetime.now() + timedelta(seconds=120)
```

Comment:
sorry if I confused you.
You can use ocs-ci/utility/retry.py on the old implementation with removing the sleep.
let me know if you need my help with enabling this retry decorator

Reply:
Thanks Raz for pointing to the retry decorator, made changes accordingly